lib/widgets/action_sheet.dart

```diff
@@ -163,6 +163,58 @@ class ActionSheetCancelButton extends StatelessWidget {
   }
 }
 
+/// Show a sheet of actions you can take on a channel.
+void showChannelActionSheet(BuildContext context, {
+  required int streamId,
```

Comment:
Let's use the modern term `channelId` instead of `streamId`. (There are still several places in the existing code that haven't migrated yet; we'll address those separately.)